indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -204,9 +204,8 @@ def _send_emails(self, form):
             template = get_template_module('events/registration/emails/custom_email.html',
                                            email_subject=email_subject, email_body=email_body)
             bcc = [session.user.email] if form.copy_for_sender.data else []
-            attachments = (get_ticket_attachments(registration)
-                           if 'attach_ticket' in form and form.attach_ticket.data
-                           else None)
+            attach_ticket = not registration.is_ticket_blocked and 'attach_ticket' in form and form.attach_ticket.data
```

Comment:
I'd move `not registration.is_ticket_blocked` to the end since it's likely more expensive than just accessing the form data.